### ISSUE TITLE
Shed requires two passes when removing imports makes it possible to put them on one line

### ISSUE THREAD

**Issue by jakkdl** [Zac-HD/shed#75]
Input:
```python
from os import (
    wait,
    wait3,
    wait4,
    waitid,
    waitid_result,
    waitpid,
    waitstatus_to_exitcode,
    walk,
    write,
    writev,
)

print(
    waitpid,
    waitstatus_to_exitcode,
    walk,
    write,
    writev,
)
```
First pass:
```python
from os import (
    waitpid,
    waitstatus_to_exitcode,
    walk,
    write,
    writev,
)

print(
    waitpid,
    waitstatus_to_exitcode,
    walk,
    write,
    writev,
)
```
second pass:
```python
from os import waitpid, waitstatus_to_exitcode, walk, write, writev

print(
    waitpid,
    waitstatus_to_exitcode,
    walk,
    write,
    writev,
)
```

Quite irritating :unamused:

**Comment by Zac-HD** [comment]
Fixed by #82 (github syntax magic just missed it)

**Comment by jakkdl** [comment]
:tada:
